appsec/src/helper/engine.cpp

```diff
@@ -190,10 +211,9 @@ template <typename T> engine::action parse_action(T &action_object)
 }
 
 template <typename T, typename>
-engine::action_map engine::parse_actions(
-    const T &doc, const engine::action_map &default_actions)
+engine::action_map engine::parse_actions(const T &doc)
 {
-    engine::action_map actions = default_actions;
+    engine::action_map actions = {};
```

Comment:
We don't need to parse actions anymore so all of this logic needs to go.

Reply:
Done